app/controllers/cart_categories_controller.rb

```diff
@@ -14,4 +14,14 @@ def index
       @total += category.price * count
     end
   end
+
+  def destroy
+    @cart.remove(params[:category_id])
+    category = Category.find(params[:category_id])
+    flash[:notice] = "Successfully removed
+      #{view_context.link_to(category.name,
+      city_categories_path(category.city_id),
+      {:style=>'color:#00FF00;', :class => "css_class"})} from your cart."
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator `=>`.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer single-quoted strings inside interpolations.